assets/DEXSeq.gff

```diff
@@ -0,0 +1,253 @@
+I	dexseq_prepare_annotation.py	aggregate_gene	335	792	.	+	.	gene_id "YAL068W-A+YAL069W"
+I	dexseq_prepare_annotation.py	exonic_part	335	537	.	+	.	gene_id "YAL068W-A+YAL069W"; transcripts "YAL069W"; exonic_part_number "001"
+I	dexseq_prepare_annotation.py	exonic_part	538	649	.	+	.	gene_id "YAL068W-A+YAL069W"; transcripts "YAL068W-A+YAL069W"; exonic_part_number "002"
+I	dexseq_prepare_annotation.py	exonic_part	650	792	.	+	.	gene_id "YAL068W-A+YAL069W"; transcripts "YAL068W-A"; exonic_part_number "003"
+I	dexseq_prepare_annotation.py	aggregate_gene	1807	2169	.	-	.	gene_id "YAL068C"
```

Comment:
Is this specific to a genome?
To a genome version?
Maybe the filename could reflect that

Reply:
This was a file leftover from testing by accident. It is no needed anymore. Fixed in #68